packages/mobile/src/index.d.ts

```diff
@@ -41,3 +41,66 @@ declare module '@ungap/url-search-params'
 declare module 'react-native-install-referrer'
 declare module 'react-native-secure-key-store'
 declare module 'qrcode'
+declare module 'web3-eth-abi' {
+  export interface ABIDefinition {
```

Comment:
Nit: could we put this into it's own file and import it here?
Also why do we need this?